src/replication.c

```diff
@@ -2173,6 +2235,211 @@ istgt_lu_resize_volume(spec_t *spec, uint64_t size) {
 	return true;
 }
 
+#define IS_REPLICA_ID_EXIST_IN_LIST(_replica_id, _list, _no_of_items, _is_exist) 	\
+	do {										\
+		int i;									\
+		for (i=0; i < _no_of_items; i++) {					\
+			if (strncmp(_replica_id, _list[i], REPLICA_ID_LEN) == 0) {	\
+					_is_exist = true;				\
+					break;						\
+			}								\
+		}									\
+	}while(0)
+
+
+#ifdef REPLICATION
```

Comment:
Not needed. Code from this file will get compiled if `REPLICATION` is defined.

Reply:
updated